### PR TITLE
Update Directory Structure and Update Native Package Name

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
Part of the [#1623](https://github.com/ballerina-platform/ballerina-standard-library/issues/1623)

## Examples
N/A

## Checklist
- [x] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -57,6 +57,7 @@ ballerina {
     packageOrganization = packageOrg
     module = packageName
     langVersion = ballerinaLangVersion
+    testCoverageParam = "--code-coverage --coverage-format=xml --includes=io.ballerina.stdlib.io.*:ballerina.io*"
```
Comment (niveathika) [inline ballerina/build.gradle:60]: Shall we bump the plugin version to `0.10.0`? This should be addressed in the latest version

Comment (ThisaruGuruge) [reply ballerina/build.gradle:60]: Ah right. Will do.

**Review by BuddhiWathsala — APPROVED** [review]

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -4,14 +4,14 @@ name = "io"
...
+path = "../native/build/libs/io-native-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "io"
 version = "@project.version@"
 
 
 [[platform.java11.dependency]]
-path = "../io-test-utils/build/libs/io-test-utils-@project.version@.jar"
+path = "../test-utils/build/libs/io-test-utils-@project.version@.jar"
 groupId = "org.ballerinalang.stdlib.io"
```
Comment (kalaiyarasiganeshalingam) [inline ballerina/Ballerina.toml:15]: Don't we need to update this?

Comment (niveathika) [reply ballerina/Ballerina.toml:15]: Since the path is given, we can actually remove the rest of the params except scope.

Comment (ThisaruGuruge) [reply ballerina/Ballerina.toml:15]: Updated with https://github.com/ballerina-platform/module-ballerina-io/pull/252

**Pull request merged**